app/assets/v2/js/pages/dashboard.js

```diff
@@ -73,7 +73,8 @@ var buildURI = function() {
   let uri = '';
   let _filters = filters.slice();
 
-  _filters.push('keywords', 'order_by', 'org');
+  _filters.push('keywords', 'order_by', 'order_by_date');
```

Comment:
is there a reason why you removed the filter for 'org' there?

Reply:
@kuhnchris sorry for that i've fixed it